lib/ansible/plugins/netconf/sros.py

```diff
@@ -19,6 +19,23 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
+DOCUMENTATION = """
+---
+netconf: sros
+short_description: Use Nokia SROS netconf plugin to run netconf commands on Nokia SROS platform
+description:
+  - This sros plugin provides low level abstraction apis for
+    sending and receiving netconf commands from Nokia sros network devices.
+version_added: "2.10"
+options:
+  ncclient_device_handler:
+    type: str
+    default: alu
```

Comment:
The ncclient device-handler for Nokia SR OS called "alu" was built years ago. From latest testing the usage of the "default" handler provides actually better results. Decision needs to be made, to either provide an updated version of the SROS handler in ncclient - or simply linking to the default handler from Ansible.